llvm/lib/Analysis/MLInlineAdvisor.cpp

```diff
@@ -187,8 +187,8 @@ unsigned MLInlineAdvisor::getInitialFunctionLevel(const Function &F) const {
   return CG.lookup(F) ? FunctionLevels.at(CG.lookup(F)) : 0;
 }
 
-void MLInlineAdvisor::onPassEntry(LazyCallGraph::SCC *LastSCC) {
-  if (!LastSCC || ForceStop)
+void MLInlineAdvisor::onPassEntry(LazyCallGraph::SCC *CurSCC) {
```

Comment:
IIUC s/LastSCC/CurSCC is a clarity refactoring, the semantics didn't change, correct?

we could just do a quick NFC change with just this renaming, to keep the change focused to just the fallout from 94815? (up to you - I don't think that precursor change should need review, and happy to do it, too, if you want)